### PR TITLE
Final questions from Kanat Kaiyp

### PR DESCRIPTION
**Pull request by kanat-kaiyp**

### CODE REVIEW

File: question_zbbb3.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+There is no single ______ function.
++exec()
+void()
+ppid()
+main()
```
Comment (konsteacher) [inline question_zbbb3.md]: What?

File: question_zRSRH.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+What does this command do?
+  head /etc/passwd
+• display whole content of a file
+• display the first 10 lines of a file
+• display nothing
+• display the first 100 lines of a file
```
Comment (konsteacher) [inline question_zRSRH.md]: Copy-paste?

File: question_zbbb3.md (hunk 1 above)
Comment (kanat-kaiyp) [inline question_zbbb3.md]: It's my mistake. When I have written question, I didn't see correctly that where is no any brackets near "exec". Text, that I have used: "There is no single exec function; instead, there is a family of exec functions built on a single system call." So, I will update this question.

File: question_zRSRH.md (hunk 2 above)
Comment (kanat-kaiyp) [inline question_zRSRH.md]: I didn't copy. I just take "head" command and put near it the first directory, which crossed my mind. It is coincidence.

**Pull request merged**